k8s/pkg/apis/longhorn/v1beta2/backingimage.go

```diff
@@ -36,16 +39,30 @@ type BackingImageDiskFileStatus struct {
 	LastStateTransitionTime string `json:"lastStateTransitionTime"`
 }
 
+type BackingImageDiskFileSpec struct {
+	// +optional
+	EvictionRequested bool `json:"evictionRequested"`
+}
+
 // BackingImageSpec defines the desired state of the Longhorn backing image
 type BackingImageSpec struct {
+	// Deprecated. We are now using DiskFileSpecMap to assign different spec to the file on different disks.
 	// +optional
 	Disks map[string]string `json:"disks"`
 	// +optional
+	DiskFileSpecMap map[string]*BackingImageDiskFileSpec `json:"diskFileSpecMap"`
+	// +optional
 	Checksum string `json:"checksum"`
 	// +optional
 	SourceType BackingImageDataSourceType `json:"sourceType"`
 	// +optional
 	SourceParameters map[string]string `json:"sourceParameters"`
+	// +optional
+	MinNumberOfCopies int `json:"minNumberOfCopies"`
```

Comment:
nit: using `MinNumberOfReplicas` might be better.

Reply:
prefer copy to avoid get confused with the volume replica